src/filemanager/filemanager.h

```diff
@@ -111,7 +111,7 @@ class FileManager : public QObject
      *
      */
     struct FolderStruct {
-        FolderStruct(QDir _dir, QString _extension = QString())
+        FolderStruct(QDir _dir, const QString& _extension = QString())
```

Comment:
I'd remove that completely. FolderStruct {dir, extension}; already creates what you want.

Reply:
LCMTH, great catch